.prettierrc

```diff
@@ -0,0 +1,5 @@
+{
+    "arrowParens": "always",
+    "singleQuote": true,
+    "trailingComma": "all"
```

Comment:
Is this the semicolon at the end? I prefer not to have it 😅

Reply:
I checked and it's a matter of adding "semi": false here, if you are ok with that

Reply:
I prefer having semicolons but I am fine with not having. Up to you here ;)
Prettier reformats everything so it is easy to use any way.